cvat-ui/src/components/analytics-report/analytics-page-header.tsx

```diff
@@ -0,0 +1,96 @@
+// Copyright (C) CVAT.ai Corporation
+//
+// SPDX-License-Identifier: MIT
+
+import React from 'react';
+import { Link } from 'react-router-dom';
+import Title from 'antd/lib/typography/Title';
+import Button from 'antd/lib/button';
+import DatePicker from 'antd/lib/date-picker';
+import { Row, Col } from 'antd/lib/grid';
+import { DownloadOutlined } from '@ant-design/icons';
+
+import { Project, Task, Job } from 'cvat-core-wrapper';
+
+interface Props {
+    onExportEvents(): void;
+    onUpdateTimePeriod(from: Date | null, to: Date | null): void;
+    resource: Project | Task | Job;
+    exporting: boolean;
+    fetching: boolean;
+}
+
+function AnaylyticsPageHeader(props: Props): JSX.Element {
+    const {
+        onUpdateTimePeriod,
+        onExportEvents,
+        resource,
+        exporting,
+        fetching,
+    } = props;
+
+    return (
+        <Row justify='space-between' align='middle'>
+            <Col className='cvat-analytics-header'>
+                <Title level={4} className='cvat-text-color'>
+                    {'Analytics page for '}
```

Comment:
```suggestion
                    {'Analytics for '}
```
Maybe we should omit `page`